src/js/api/config.js

```diff
@@ -10,7 +10,6 @@ define([
         //androidhls: true,
         autostart: false,
         controls: true,
-        cookies: true,
```

Comment:
@androoP @esboyd ? Is this feature no longer needed if the settings are stored in local storage?